src/components/TextareaField/index.ts

```diff
@@ -0,0 +1,3 @@
+import TextAreaField from './TextAreaField';
```

Comment:
Or did you rename the file path? The diff on GitHub doesn't say so - maybe you have a case-insensitive file-system.

Reply:
The filesystem is case-sensitive, and should be updated.